src/styles/block/contacts.scss

```diff
@@ -0,0 +1,91 @@
+.contacts {
+  &__container {
+    @include onTablet {
+      display: grid;
+      column-gap: 24px;
+      grid-template-columns: repeat(6, 1fr);
+    }
+
+    @include onDesktop {
+      display: grid;
+      column-gap: 24px;
+      grid-template-columns: repeat(12, 1fr);
+    }
+
+    &--info {
+      @include onTablet {
+        grid-column: 1 / 4;
+      }
+
+      @include onDesktop {
+        grid-column: 1 / 7;
+
+        display: grid;
+        column-gap: 24px;
+        column-width: 63px;
+
+        grid-template-columns: repeat(6, 1fr);
```

Comment:
Why do you repeat `display: grid;` everywhere? Use it for once for the element itself. Or better create a universal `grid` system.